### PR TITLE
Create celestials

### PR DESCRIPTION
**Pull request by MarcelSF**

### CODE REVIEW

File: app/controllers/celestials_controller.rb [hunk 1]
```diff
@@ -1,2 +1,35 @@
 class CelestialsController < ApplicationController
+  def new
+    @celestial = Celestial.new
+  end
+
+  def create
+    @celestial = Celestial.new(celestial_params)
+    puts @celestial.errors.full_messages
```
Comment (sensotape) [inline app/controllers/celestials_controller.rb]: Can we get rid of puts in controller?

File: app/views/celestials/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<%= simple_form_for celestial do |f| -%>
```
Comment (sensotape) [inline app/views/celestials/_form.html.erb]: Can you please reindent this document?

Comment (sensotape) [reply app/views/celestials/_form.html.erb]: And why do we need divs there?

File: app/controllers/celestials_controller.rb (hunk 1 above)
Comment (MarcelSF) [reply app/controllers/celestials_controller.rb]: Yes, no problem :)

**Pull request merged**